tests/drivers/spi/spi_loopback/boards/96b_stm32_sensor_mez.conf

```diff
@@ -0,0 +1,5 @@
+CONFIG_SPI_4=y
```

Comment:
no need to add that one since you already made it default on board's kconfig.defconfig

Reply:
Fixed.